### PR TITLE
WIP #20 Windows gotchas

### PR DESCRIPTION
**Pull request by oconnelc**
#20 Description

This commit contains all of the gotchas that I found while setting up a Windows dev environment s part of the effort for issue #20 

### Type of change
- Documentation update

### How Has This Been Tested?
N/A

### Screenshots
N/A

### CODE REVIEW

File: NotesForRunningOnWindows.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+1:13:22 PM web.1     |  operable program or batch file.
+[DONE] Killing all processes with signal  SIGINT
+1:13:22 PM webpack.1 |  'bin' is not recognized as an internal or external command,
+1:13:22 PM webpack.1 |  operable program or batch file.
+1:13:22 PM web.1     Exited with exit code null
+1:13:22 PM webpack.1 Exited with exit code null
+1:13:22 PM sidekiq.1 Exited with exit code SIGINT
+```
+
+Running the commandds inside of the Procfile.dev seems to be a good workaround. s
```
Comment (h-m-m) [inline NotesForRunningOnWindows.md]: Minor typo here

Comment (h-m-m) [inline NotesForRunningOnWindows.md]: Minor typos here

File: README.md [hunk 2]
```diff
@@ -132,16 +132,17 @@ $ yarn install
...
+visit https://devcenter.heroku.com/articles/heroku-cli for instructions to install Heroku.
+```
+# Viewing the Application
+## if you chose the local route, then you are good to go on:
   http://localhost:5000
   $ rspec (to run the test suite) or bundle exec rspec (if the first does not work)
-# If you chose the Docker route:
-  $ docker-compose up -d -or- $ docker-compose up
+## If you chose the Docker route:
+  ```$ docker-compose up -d -or- $ docker-compose up```
```
Comment (h-m-m) [inline README.md]: I think the `-or-` should be outside the code block here

File: README.md [hunk 3]
```diff
@@ -132,16 +132,17 @@ $ yarn install
 $ rake dev:setup
 $ heroku local -f Procfile.dev
 # if you run into the following error:
-```
```
Comment (h-m-m) [inline README.md:135]: Maybe the better fix here is moving what's in comments in the readme out into plain text. Something like:

```
$ heroku local -f Procfile.dev
```
if you run into the error `bash: heroku: command not found` visit [link]

**Review by h-m-m — COMMENTED** [review]
Thank you so much for spending time on this! Because we regularly get Windows users who show up to volunteer, I like working to make sure they feel welcome. I'm so happy you've worked on improving that experience.

I have a few suggestions, but they're mostly aimed at minor readability issues.

**Review by h-m-m — APPROVED** [review]
These are great edits, and they definitely match what I saw from helping people last week.

I also like your improvement and change of header levels on the main readme. I'm sure there's more we can do to make the whole process smoother, but that alone should clear up a lot of confusion.

**Review by maebeale — APPROVED** [review]

**Pull request merged**